### ISSUE TITLE
Compile errors

### ISSUE THREAD

**Issue by devlo** [nordlow/gmp-d#3]
Hi, I have problem using this library on Ubuntu 16.04:

Fetching libgmp 1.0.0 (getting selected version)...
Fetching gmp-d 0.0.1 (getting selected version)...
Performing "debug" build using ldc2 for x86_64.
libgmp 1.0.0: building configuration "library"...
gmp-d 0.0.1: building configuration "library"...
../../.dub/packages/gmp-d-0.0.1/gmp-d/src/gmp.d(192,43): Error: no property '_ccc' for type 'MpZ'
../../.dub/packages/gmp-d-0.0.1/gmp-d/src/gmp.d(189,9): Error: function gmp.MpZ.dup no return exp; or assert(0); at end of function
memberFun:non-ref this
../../.dub/packages/gmp-d-0.0.1/gmp-d/src/gmp.d(1144,63): Error: undefined identifier '_ccc'
ldc2 failed with exit code 1.

**Comment by nordlow** [comment]
Does it happen on the master branch of gmp-d aswell?

I'm gonna bump the version of gmp-d to 0.0.2.

**Comment by devlo** [comment]
gmp-d ~master: building configuration "library"...
../../.dub/packages/gmp-d-master/gmp-d/src/gmp/z.d(249,13): Error: function gmp.z.MpZ.fromString functions cannot be scope
../../.dub/packages/gmp-d-master/gmp-d/src/gmp/z.d(1125,19): Error: function gmp.z.MpZ._limbs functions cannot be scope
../../.dub/packages/gmp-d-master/gmp-d/src/gmp/z.d(1156,26): Error: function gmp.z.MpZ._ptr functions cannot be scope
../../.dub/packages/gmp-d-master/gmp-d/src/gmp/q.d(350,26): Error: function gmp.q.MpQ._ptr functions cannot be scope
../../.dub/packages/gmp-d-master/gmp-d/src/gmp/q.d(356,26): Error: function gmp.q.MpQ._num_ptr functions cannot be scope
../../.dub/packages/gmp-d-master/gmp-d/src/gmp/q.d(362,26): Error: function gmp.q.MpQ._den_ptr functions cannot be scope

**Comment by nordlow** [comment]
There's a serious regression in 2.074.0:

```
dub build --compiler=/usr/bin/dmd
```

errors as

```
Performing "debug" build using /usr/bin/dmd for x86_64.
gmp-d 0.0.1+commit.126.g660d82b: building configuration "library"...
src/gmp/z.d(484,20): Error: struct gmp.z.MpZ is not copyable because it is annotated with @disable
src/gmp/q.d(205,16): Error: template instance gmp.z.MpZ.opBinary!"/" error instantiating
src/gmp/z.d(168,16): Error: struct gmp.z.MpZ is not copyable because it is annotated with @disable
src/gmp/z.d(921,16): Error: struct gmp.z.MpZ is not copyable because it is annotated with @disable
/usr/bin/dmd failed with exit code 1.
```

I don't, currently, know how to move forward with this.

I asked the forums [here](http://forum.dlang.org/thread/pscljqsdhgyylzvpisnb@forum.dlang.org).

**Update:**

This error message seems to be a false positive triggered by not being able to handle my scope tagging of functions which I just commented out.

**Comment by nordlow** [comment]
I just removed the `scope` qualifiers for functions and now build and tests pass for dmd latest stable and master. Available on gmp-d master and new tag v0.0.2.
